app/subscriber/src/components/media-type-filters/FilterOptions.tsx

```diff
@@ -27,78 +40,78 @@ export const HomeFilters: React.FC<IHomeFilterProps> = () => {
     mediaTypeIds: [],
   };
 
-  const handleFilterClick = (type: HomeFilterType) => {
+  const handleFilterClick = (type: FilterOptionTypes) => {
```

Comment:
@kylgarmor - this is where that enum is being used (it was poorly named MediaTypeFilter by me), but as you can see it is its own beast with `contentType`, `mediaTypeId`, and `source`. Spoke with Jeremy and for now we are going to live with it as the grouping does not line up with anywhere else in the app. Renamed to `FilterOptionTypes` for lack of a better name